### PR TITLE
vochain/scrutinizer: port most of vote results to sqlite

### PR DESCRIPTION
**Pull request by mvdan**
(see commit message)

### CODE REVIEW

**Comment by mvdan** [comment]
@p4u FYI this keeps the existing design when it comes to updating the existing processes with some computation, e.g. WalkEnvelopes, but it's very likely that we could make some of it more efficient with sqlite by writing SQL queries with logic in them. That way we would avoid some of the data encoding and decoding cost. We can talk about that once the sqlite implementation is complete though, but still FYI.

File: vochain/scrutinizer/db/process.sql.go [hunk 1]
```diff
@@ -94,11 +101,12 @@ func (q *Queries) CreateProcess(ctx context.Context, arg CreateProcessParams) (s
 		arg.CreationTime,
 		arg.SourceBlockHeight,
 		arg.SourceNetworkID,
+		arg.ResultsVotes,
 	)
 }
 
 const getProcess = `-- name: GetProcess :one
-SELECT id, entity_id, entity_index, start_block, end_block, results_height, have_results, final_results, census_root, rolling_census_root, rolling_census_size, max_census_size, census_uri, metadata, census_origin, status, namespace, envelope_pb, mode_pb, vote_opts_pb, private_keys, public_keys, question_index, creation_time, source_block_height, source_network_id FROM processes
+SELECT id, entity_id, entity_index, start_block, end_block, results_height, have_results, final_results, census_root, rolling_census_root, rolling_census_size, max_census_size, census_uri, metadata, census_origin, status, namespace, envelope_pb, mode_pb, vote_opts_pb, private_keys, public_keys, question_index, creation_time, source_block_height, source_network_id, results_votes, results_weight, results_envelope_height, results_signatures, results_block_height FROM processes
```
Comment (jordipainan) [inline vochain/scrutinizer/db/process.sql.go:109]: Is this not the same as `SELECT * FROM` ?

**Comment by jordipainan** [comment]
LGTM! Thank you @mvdan 

Fyi: I made a comment but the code is autogenerated so does not apply and I deleted it, in case some mail notification arises :)

**Comment by p4u** [comment]
Great @mvdan, we are close to finally remove badgerhold.

**Review by p4u — APPROVED** [review]

**Pull request merged**